staking/integration-tests/src/integrity_pool/instructions.rs

```diff
@@ -175,12 +175,16 @@ pub fn advance_delegation_record(
     stake_account_positions: Pubkey,
     pyth_token_mint: Pubkey,
     pool_data: Pubkey,
+    publisher_stake_account_positions: Option<Pubkey>,
 ) -> TransactionResult {
     let delegation_record = get_delegation_record_address(publisher, stake_account_positions);
     let custody_addess = get_pool_reward_custody_address(pyth_token_mint);
     let pool_config_pubkey = get_pool_config_address();
     let stake_account_custody = get_stake_account_custody_address(stake_account_positions);
 
+    let publisher_stake_account_custody =
+        publisher_stake_account_positions.map(get_stake_account_custody_address);
```

Comment:
haha weird way to call the function

Reply:
This was the easiest way to call it on an Option to also get an Option